packages/zowe-explorer/src/trees/shared/SharedActions.ts

```diff
@@ -187,13 +187,21 @@ export class SharedActions {
             if (pattern.indexOf("/") > -1) {
                 // USS file was selected
                 const filePath = pattern.substring(pattern.indexOf("/"));
-                const sessionNode = ussTree.mSessionNodes.find((sessNode) => sessNode.getProfileName() === sessionName);
+                const sessionNode = ussTree.mSessionNodes.find((sessNode) => sessNode.label.toString().toLowerCase() === sessionName.toLowerCase());
+                if (!sessionNode) {
+                    Gui.showMessage(vscode.l10n.t("Profile not found."));
```

Comment:
In the case that the profile is not found, should we delete the invalid recent entry from the list? Perhaps we could also add a prompt to the message asking the user if they want to keep or delete the invalid entry.

Just an idea, not a request changes for this PR - we could address this as a future enhancement 😋